api/v1/routes/payment.py

```diff
@@ -0,0 +1,24 @@
+from fastapi import APIRouter
+from api.v1.services.payment import PaymentService
+from api.db.database import get_db
+from sqlalchemy.orm import Session
+from fastapi import Depends, status
+from api.v1.models.user import User
+from api.v1.services.user import user_service
+from api.v1.schemas.payment import CreatePaymentSchema
+
+payment = APIRouter(prefix='/payments', tags=['Payment'])
+
+
+
+@payment.post('/create', status_code=status.HTTP_201_CREATED)
+async def post_single_payments(request: CreatePaymentSchema, db: Session = Depends(get_db), current_user: User = Depends(user_service.get_current_user)):
```

Comment:
Change request to `payment_schema`

Reply:
Pls clarify the exact change needed for `payment-schema`

Reply:
this has been resolved @joboy-dev